avocado/plugins/yaml_to_mux.py

```diff
@@ -262,8 +268,12 @@ def run(self, args):
             try:
                 args.mux.data_merge(create_from_yaml(multiplex_files, debug))
             except IOError as details:
-                logging.getLogger("avocado.app").error(details.strerror)
-                sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+                if getattr(args, "mux_ignore_missing", 'off') == 'on':
+                    msg = 'Cannot access the multiplex file(s) in this host.'
```

Comment:
Well I kind of like this option, but I guess users could add the local files as a workaround.